app/tests/integration/nimbus/pages/browser/__init__.py

```diff
@@ -8,3 +13,33 @@ def execute_script(selenium, *args, script=None, context=None):
                 return selenium.execute_script(script, *args)
         else:
             return selenium.execute_script
+
+
+class AboutConfig(Page):
+
+    URL_TEMPLATE = "about:config"
+
+    _search_bar_locator = (By.ID, "about-config-search")
+    _row_locator = (By.CSS_SELECTOR, "tr > td > span > span")
+
+    def __init__(self, selenium, **kwargs):
+        super().__init__(selenium, timeout=80, **kwargs)
+
+    def wait_for_page_to_load(self):
+        self.wait.until(EC.presence_of_element_located(self._search_bar_locator))
+        return self
+
+    def wait_for_pref_flip(self, pref=None):
+        timeout = time.time() + 60 * 5
+        while time.time() < timeout:
+            try:
+                search_bar = self.find_element(*self._search_bar_locator)
+                search_bar.send_keys("nimbus.qa.pref-1")
```

Comment:
Oh maybe pull this out into a constant or an optional parameter with a default value or something just so it's clear which pref it's talking about?

Reply:
ah yes, forgot about that one!